package.json

```diff
@@ -29,6 +29,7 @@
     "axios": "^1.2.1",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
+    "console": "^0.7.2",
```

Comment:
why do we need this?

Reply:
![image](https://github.com/defientco/CRE8ORS-UI/assets/127037083/9f264efc-bd72-4afa-a962-df746b5dc81a)
In many code, I can see importing part about this lib.

Reply:
removed 🚮